.github/workflows/tests.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: deps
-          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
```

Comment:
If you're updating this, you should update the `local.rebar` and `local.hex` steps to happen based on the ASDF cache rather than the Elixir cache.

Reply:
Yeah I actually got rid of this change and did that instead, it makes more sense. Thanks for the suggestion.